docs/ref/models/relations.txt

```diff
@@ -110,7 +120,14 @@ Related objects reference
         needed. You can use callables as values in the ``through_defaults``
         dictionary.
 
+    .. versionchanged:: 4.1
```

Comment:
Calling this out because this might not be correct protocol, but `acreate` was included in the [4.1 release](https://docs.djangoproject.com/en/4.1/releases/4.1/#asynchronous-orm-interface) but not called out in the docs until now

Reply:
It was added but unintentionally, see ticket-34139.

Reply:
Ah I see. I’ll revert this change when I get back to my computer